Modules/posixmodule.c

```diff
@@ -4471,9 +4446,32 @@ os__path_splitroot_impl(PyObject *module, path_t *path)
         *p = L'\\';
     }
 
+#if MS_WINDOWS_GAMES
+    /* network share */
+    if (buffer[0] == L'\\' && buffer[1] == L'\\') {
+        end = buffer + 2;
+        for (int i = 0; i < 2; ++i) {
+            end = wcschr(end, '\\');
+            if (end == NULL) {
+                break;
+            }
+            end++;
+        }
+        ret = (end != NULL) ? S_OK : E_INVALIDARG;
+    }
+    /* Check for absolute drive path */
```

Comment:
It doesn't have to be an absolute path. For example, `"C:spam\\eggs"` splits as `('C:', 'spam\\eggs')`. The 'C:' component resolves to the working directory on the drive, which includes at least the root directory.

Reply:
This should be fixed.